main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_data:
+        players = json.load(players_data)
+
+    for player in players.keys():
+        player_instance = Player.objects.create(
+            nickname=player,
+            email=players[player]["email"],
+            bio=players[player]["bio"],
+            race=Race.objects.get(name=players[player]["race"]["name"])
+            if Race.objects.filter(name=players[player]["race"]["name"])
+            else Race.objects.create(
+                name=players[player]["race"]["name"],
+                description=players[player]["race"]["description"] if
+                players[player]["race"]["description"] else None
```

Comment:
I think it has bad readability when you use a lot of conditions and create another object inside object creation, create the race outside and then use it

Reply:
@Serhii-Leonenko Solution updated, please, have a look at it.

Reply:
@Serhii-Leonenko Hello, please, have a look at the solution [0cf598b](https://github.com/mate-academy/py-game-models/pull/193/commits/0cf598ba8e3c95767a16de27cdc18f9c4851e960)

Reply:
@Serhii-Leonenko , please, check the task.